components/04-views/catalog/grid/grid.hbs

```diff
@@ -2,20 +2,26 @@
 
 <main class="container">
     {{ render '@breadcrumbs' }}
-
-    <ul class="catalog-grid">
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-        {{ render '@catalog-grid-item' product }}
-    </ul>
-
-    {{ render '@pager' }}
+    <div class="catalog-list">
+        <div class="catalog-list__sidebar">
+            {{ render '@filters' }}
+        </div>
+        <div class="catalog-list__products">
+            {{ render '@toolbar' }}
+            <ul class="catalog-grid">
+                {{ render '@catalog-grid-item' product merge=true }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+                {{ render '@catalog-grid-item' product }}
+            </ul>
+            {{ render '@pager' }}
```

Comment:
can we move pager outside `catalog-list__products`?

Reply:
It's better to leave it here to fit pager in grid